kubernetes/container/build.go

```diff
@@ -1,4 +1,4 @@
-package container
+package containers
```

Comment:
If we are changing the package name then the file name should also we have to change, like:
`kubernetes/container/build.go` -> `kubernetes/containers/build.go`

Reply:
Made the changes @chandankumar4